README.md

```diff
@@ -1,2 +1,21 @@
-# ppa-nlp
-Text mining the stories of poetic forms
+# `corppa`  PPA full-text corpus utilities
+
+This repository provides code and other resources associated with the [Princeton Prosody Archive](https://prosody.princeton.edu/) (PPA), with a  particular focus on working with the PPA full-text corpus.
+
+## Development instructions
+
+This repo uses [git-flow](https://github.com/nvie/gitflow) branching conventions; **main** contains the most recent release, and work in progress will be on the **develop** branch. Pull requests for new features should be made against develop.
+
+### Developer setup and installation
+
+- **Recommended:** create a python virtual environment with your tool of choice (virtualenv, conda, etc); use python 3.10 or higher
+
+- Install the local checked out version of this package in editable mode (`-e`), including all python dependencies  and optional dependencies for development and testing:
+```sh
+pip install -e .[dev]
```

Comment:
Add the quotes so that this will play nicely with both macOS and Windows

```suggestion
pip install -e ".[dev]"
```

Reply:
oh, good catch! yeah, that's actually necessary in my shell too